docs/docs/dart_api/dataclass.md

```diff
@@ -1,23 +1,112 @@
 ---
 
-title: Custom row classes
-description: Use your own classes as data classes for drift tables
+title: Dataclass
```

Comment:
I'm still not too happy with the generic `Dataclass` name, does "Generated classes" with a description like "Overview of classes generated by drift for reading and writing to the database" work?